composer.json

```
@@ -186,7 +186,7 @@
         "sylius/user-bundle": "self.version"
     },
     "conflict": {
-        "doctrine/orm": ">= 2.15.2",
+        "doctrine/orm": ">= 2.16.0",
```

```suggestion
        "doctrine/orm": "2.16.0",
```
As there is a bug reported on doctrine/orm, maybe we should add a conflict only for 2.16.0 for now, so that, once repaired, we are not left with unnecessary conflict

---

The last 4 releases were conflicting, so let's keep it as it is.